media/css/press/press-ie.less

```diff
@@ -0,0 +1,4 @@
+.tooltip {
+    margin-bottom: 20px;
+    max-width: 75%;
+}
```

Comment:
So the tooltip seems to overlap the following field label a bit in modern browsers as well. This is in dev edition:

<img width="367" alt="screen shot 2015-10-13 at 4 13 40 pm" src="https://cloud.githubusercontent.com/assets/317532/10468581/72af24b4-71c5-11e5-8db7-889d283e2b21.png">

Looks like instead of making an IE specific CSS file & bundle, we could just move these rules into the main `press.less`? Or add a bit of top margin to `form.press-inquiry` (since that selector already exists)?